FFXIVClientStructs/FFXIV/Client/UI/RaptureAtkModule.cs

```diff
@@ -108,6 +108,11 @@ public unsafe partial struct RaptureAtkModule {
     [MemberFunction("E8 ?? ?? ?? ?? EB ?? 45 33 C9 49 8D 56 ?? 41 B0 01")]
     public partial void OpenDawnStory(uint contentFinderCondition);
 
+    [MemberFunction("E8 ?? ?? ?? ?? 48 89 9B ?? ?? ?? ?? 48 8B CE")]
+    public partial void OpenSatisfactionSupply(nint a1, uint satisfactionNPC, bool a3);
+
+    public void OpenSatisfactionSupply(uint satisfactionNPC) => OpenSatisfactionSupply(nint.Zero, satisfactionNPC, false);
```

Comment:
```suggestion
    public void OpenSatisfactionSupply(uint satisfactionNPC) => OpenSatisfactionSupply(nint.Zero, satisfactionNPC, true);
```

The only case where this might be false is from a lua call, the rest of the game expects this to be true.